pypeit/reduce.py

```diff
@@ -934,23 +938,23 @@ def find_objects_pypeline(self, image, std_trace=None,
         # create the ouptut image for skymask
         skymask = np.zeros_like(image, dtype=bool)
 
-        # TODO: Figure out where to do this properly
-        spec_min = np.zeros(self.tslits_dict['nslits'], dtype=int) \
-                        if self.tslits_dict['spec_min'] is None else self.tslits_dict['spec_min']
-        spec_max = np.full(self.tslits_dict['nslits'], self.tslits_dict['nspec']-1, dtype=int) \
-                        if self.tslits_dict['spec_max'] is None else self.tslits_dict['spec_max']
-
         plate_scale = self.spectrograph.order_platescale(self.order_vec, binning=self.binning)
         inmask = self.sciImg.mask == 0
         # Find objects
         specobj_dict = {'setup': self.setup, 'slitid': 999, #'orderindx': 999,
                         'det': self.det, 'objtype': self.objtype, 'pypeline': self.pypeline}
+
```

Comment:
I don't agree with this. If the user wants to tweak the slits, then they tweak the slits and those are the slits. If they don't want to tweak the slits, then don't tweak the slits. There is no need to have an option to use the old edges unless we were going to make it an option to use old edges for some slits and tweaked edges for others in the same reduction. That would be a bookeeping nightmare and it would be better to just make the tweaking more robust. The tweaking is incidentally extremely robust. Please make a compelling argument for this option, or simply remove it from the object finding. 

I may need the original edges for some things (like flexure compensation, i.e. there I need the box stars) but we don't need them for object finding.

Reply:
Let's talk about this at a meeting.  I prefer my approach, but understand where you're coming from.  To be clear, the `select_edges` method is used many places and isn't specific to object finding.